packages/lib-utils/src/components/details-page-header/utils/Breadcrumbs.tsx

```diff
@@ -0,0 +1,51 @@
+import { v4 as uuidv4 } from 'uuid';
+import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export type BreadcrumbProp = { name: string; path: string; uuid?: string; id?: string };
```

Comment:
I'm not sure if uuid should be exposed on the props - more so an internal fallback if no valid ID is provided